### DISCUSSION TITLE
LFX Mentorship 2022 03-Sept-Nov Challenge - For #1742

### DISCUSSION THREAD

**Discussion by hydai** [WasmEdge/WasmEdge#1752] [Show and tell]
### Introduction

For those who want to apply for LFX mentorship for #1742, this is a pretest for the application.
To complete this challenge, please follow [this guide](https://wasmedge.org/book/en/dev/rust/wasinn.html) for setting up the working environment and run [the examples](https://github.com/second-state/WasmEdge-WASINN-examples/tree/master/openvino-mobilenet-image).

### Requirements

1. Create a repository. There must be a README file to show how you finish the setup and some screenshots of the examples' execution results.
2. Pickup a Linux distribution as the basic environment.
3. Set up the working environment with [this guide](https://wasmedge.org/book/en/dev/rust/wasinn.html).
4. Run [this example](https://github.com/second-state/WasmEdge-WASINN-examples/tree/master/openvino-mobilenet-image) and try to replace the input image.
5. Screenshot the result of execution result.
6. Post your repo's GitHub link as a comment under this discussion.
7. [Optional]: It will be a bonus if you replace the `mobilenet` model with others. 

### Deadline

Please refer to [the timeline of LFX mentorship 2022 03-Sept-Nov](https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/03-Sept-Nov#timeline).
The final date of valid submissions will be August 31, 5:00 PM PDT

Good luck!

**Comment by spencergotowork** [comment]
Hi, I'm Ran. And this is my [repo link](https://github.com/spencergotowork/wasmEdge/tree/main/openvino-mobilenet-image).I ran the Rust code and plan to try other models in the future.
